core/test/qa_Scheduler.cpp

```diff
@@ -704,11 +704,6 @@ const boost::ut::suite<"SchedulerTests"> SchedulerTests = [] {
         scheduler.timeout_ms               = 100U; // also dynamically settable via messages/block interface
         scheduler.timeout_inactivity_count = 10U;  // also dynamically settable via messages/block interface
 
-        expect(scheduler.graph().reconnectAllEdges());
```

Comment:
Why are these checks removed?

Reply:
For life-time reasons, the `scheduler.graph() -> const Graph&` returns a const-reference that only the scheduler can (and should) execute/change. Access to the interna is neat for testing but opens the door for unsafe user-level API access. I thus changed that.